compose/compose-file/index.md

```diff
@@ -764,6 +764,27 @@ Logging options are key-value pairs. An example of `syslog` options:
     options:
       syslog-address: "tcp://192.168.0.42:123"
 
+To limit the amount of logs stored there are two options key-value pairs:
```

Comment:
We should be careful documenting this as-is, because these options depend on the logging driver that's in use. For example, `max-file` and `max-size` are only used for the default (`json-file`) logging driver.

I _do_ think this information is useful but;

- we should point out that these options are specific to that driver
- more clearly mention that these options _differ_ per driver, and refer to https://docs.docker.com/engine/admin/logging/overview/ - I see it's being referred to above as well